.github/workflows/php-coding-standards.yml

```diff
@@ -28,6 +28,8 @@ jobs:
       - name: Prepare PHP
         uses: woocommerce/grow/prepare-php@actions-v1
         with:
+        # TODO: Update to PHP8.2 once WPCS (3.0) and woocommerce-sniffs (> 0.1.3) are updated to latest versions.
+          php-version: 8.0
```

Comment:
👍 Thank you for the comments! I think this is useful because when someone check the `php-version` in the future, they can see the `TODO` easily and then act on it. 🙂